circleci: failed tests: TestErrcheck TestReturnCheck

The following test appears to have failed:

[#20673](https://circleci.com/gh/cockroachdb/cockroach/20673):

```
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:499:14: cannot use "could not write string: %v" (constant of type string) as golang.org/x/net/context.Context value in argument to log.Fatalf: missing method Deadline
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:499:44: cannot use err (variable of type error) as string value in argument to log.Fatalf
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:503:14: cannot use "could not write string: %v" (constant of type string) as golang.org/x/net/context.Context value in argument to log.Fatalf: missing method Deadline
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:503:44: cannot use err (variable of type error) as string value in argument to log.Fatalf
error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/cli
--- FAIL: TestErrcheck (0.00s)

=== RUN TestForbiddenImports
--- PASS: TestForbiddenImports (0.00s)
=== RUN TestGofmtSimplify
--- PASS: TestGofmtSimplify (0.00s)
=== RUN TestGoimports
--- PASS: TestGoimports (0.00s)
=== RUN TestGolint
--- PASS: TestGolint (0.00s)
=== RUN TestGrpc
--
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:494:56: cannot use err (variable of type error) as string value in argument to log.Fatalf
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:499:14: cannot use "could not write string: %v" (constant of type string) as golang.org/x/net/context.Context value in argument to log.Fatalf: missing method Deadline
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:499:44: cannot use err (variable of type error) as string value in argument to log.Fatalf
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:503:14: cannot use "could not write string: %v" (constant of type string) as golang.org/x/net/context.Context value in argument to log.Fatalf: missing method Deadline
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:503:44: cannot use err (variable of type error) as string value in argument to log.Fatalf
--- FAIL: TestReturnCheck (0.00s)

=== RUN TestSafeSQL
--- PASS: TestSafeSQL (0.00s)
=== RUN TestSyncMutex
--- PASS: TestSyncMutex (0.00s)
=== RUN TestTabsInShellScripts
--- PASS: TestTabsInShellScripts (0.00s)
=== RUN TestTimeutil
--- PASS: TestTimeutil (0.00s)
=== RUN TestUnused
--
/go/src/github.com/cockroachdb/cockroach/cli/cli_test.go:503:44: cannot use err (variable of type error) as string value in argument to log.Fatalf
2016/07/28 16:27:02 could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/cli
--- PASS: TestVarcheck (0.00s)
=== RUN TestVet
--- PASS: TestVet (0.00s)
FAIL check-style 0.000s
Makefile:189: recipe for target 'check' failed
make: *** [check] Error 1

```
Please assign, take a look and update the issue accordingly.

--- a/cli/cli_test.go
+++ b/cli/cli_test.go
@@ -484,37 +484,38 @@ func Example_max_results() {
 func Example_zone() {
 	c := newCLITest()
 	defer c.stop()
-	zone1 := "zone1.yaml"
-	zone2 := "zone2.yaml"
+
+	ctx := context.Background()
+
 	defer func() {
-		if err := os.Remove(zone1); err != nil {
-			log.Fatalf("could not remove zone config file: %v", err)
+		if err := os.Remove("zone1.yaml"); err != nil {
+			log.Fatalf(ctx, "could not remove zone config file: %v", err)
 		}
-		if err := os.Remove(zone2); err != nil {
-			log.Fatalf("could not remove zone config file: %v", err)
+		if err := os.Remove("zone2.yaml"); err != nil {
+			log.Fatalf(ctx, "could not remove zone config file: %v", err)
 		}
 	}()
 
-	if err := ioutil.WriteFile(zone1, []byte("replicas:\n- attrs: [us-east-1a,ssd]\n"), 0777); err != nil {
-		log.Fatalf("could not write string: %v", err)
+	if err := ioutil.WriteFile("zone1.yaml", []byte("replicas:\n- attrs: [us-east-1a,ssd]\n"), 0777); err != nil {
+		log.Fatalf(ctx, "could not write string: %v", err)
 	}
 
-	if err := ioutil.WriteFile(zone2, []byte("range_max_bytes: 134217728\n"), 0777); err != nil {
-		log.Fatalf("could not write string: %v", err)
+	if err := ioutil.WriteFile("zone2.yaml", []byte("range_max_bytes: 134217728\n"), 0777); err != nil {
+		log.Fatalf(ctx, "could not write string: %v", err)
 	}
 
 	c.Run("zone ls")
 	// Call RunWithArgs to bypass the "split-by-whitespace" arg builder.
-	c.RunWithArgs([]string{"zone", "set", "system", "--file=" + zone1})
+	c.RunWithArgs([]string{"zone", "set", "system", "--file=zone1.yaml"})
 	c.Run("zone ls")
 	c.Run("zone get system.nonexistent")
 	c.Run("zone get system.lease")
-	c.RunWithArgs([]string{"zone", "set", "system", "--file=" + zone2})
+	c.RunWithArgs([]string{"zone", "set", "system", "--file=zone2.yaml"})
 	c.Run("zone get system")
 	c.Run("zone rm system")
 	c.Run("zone ls")
 	c.Run("zone rm .default")
-	c.RunWithArgs([]string{"zone", "set", ".default", "--file=" + zone2})
+	c.RunWithArgs([]string{"zone", "set", ".default", "--file=zone2.yaml"})
 	c.Run("zone get system")
 
 	// Output:

